docs/source/support/release-notes.rst

```diff
@@ -23,7 +23,11 @@ SDK tools
 DAML
 ~~~~
 
+- **BREAKING CHANGE - DAML Compiler**: It is now an error to omit method bodies in class ``instance``s if the method
```

Comment:
```suggestion
- **BREAKING CHANGE - DAML Compiler**: It is now an error to omit method bodies in class ``instance`` s if the method
```
I _think_ this might fix the error that you are currently getting on CI: `release-notes.rst:26: WARNING: Inline literal start-string without end-string.`

Reply:
Thanks!